openreview/tools.py

```diff
@@ -1798,11 +1800,14 @@ def get_neurips_profile_info(profile, n_years=3):
 
     ## Emails section
     for email in profile.content['emails']:
+        if email.startswith("****@"):
+            raise openreview.OpenReviewException("You do not have the required permissions as some emails are obfuscated. Please login with the correct account or contact support.")
         emails.add(email)
-    ## if institution section is empty, add email domains
-    if not domains:
-        for email in profile.content['emails']:
-            domains.update(openreview.tools.subdomains(email))
+        domains.update(openreview.tools.subdomains(email))
+    # ## if institution section is empty, add email domains
+    # if not domains:
```

Comment:
Why do you remove this?

Reply:
Not completely removed, just removed the `if not domains:` condition. It'll update the domains irrespective of whether there was institution section or not.